src/features_creator/features_creator.py

```diff
@@ -38,3 +38,54 @@ def get_matching_column_names(data, pattern):
 
     """
     ...
+
+
+def calculate_percentage_change(
+    df, pattern, period=(2, 2), filter=None, changed_name=None
+):
+    """Calculate percentage change over a time period
+    (months over months or weeks over weeks)
+    for the given column pattern.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        A pandas dataframe
+    pattern : str
```

Comment:
This `pattern` is not consistent with the way that `calculate_standard_deviation` is designed. Since that function takes a list of column names, I think this one should too.

Reply:
Alternatively, we can change `calculate_standard_deviation` to take a pattern

Reply:
To my mind, it would be better to pass the pattern inside `calculate_standard_deviation` function and use `get_matching_columns`.  We might be using this function for several column patterns and it would not be feasible to type all the column names as strings or lists.